Add Python 2.7 deprecation notice and adjust package properties to require 3.6 (#1007)
stripe/stripe-python: README.md

### Requirements

- Python 3.6+ (PyPy supported)

#### Python 2.7 deprecation

> [The Python Software Foundation (PSF)](https://www.python.org/psf-landing/) community [announced the end of support of Python 2](https://www.python.org/doc/sunset-python-2/) "as of 01 January 2020, no new bug reports, fixes, or changes will be made to Python 2, and Python 2 is no longer supported."

Reviewer: ```suggestion
[The Python Software Foundation (PSF)](https://www.python.org/psf-landing/) community [announced the end of support of Python 2](https://www.python.org/doc/sunset-python-2/) on 01 January 2020.
```
nit: The quote flows kind of strangely with the beginning of the sentence IMO. I think it's fine just to say they stopped supporting it on X date.

Starting with version 6.0.0 Stripe SDK Python packages will no longer support Python 2.7. To continue to get new features and security updates, please make sure to update your Python runtime to Python 3.6+.

> The last version of Stripe SDK that supports Python 2.7 is 5.5.0.

Reviewer: ```suggestion
The last version of the Stripe SDK that supports Python 2.7 is 5.5.0.
```
